feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/balance/list/BalanceListViewModel.kt

```diff
@@ -194,6 +207,17 @@ class BalanceListViewModel(
         router.openAssetSearch()
     }
 
+    fun walletConnectClicked() {
+        launch {
+            if (walletConnectAccountSessions.first() > 0) {
```

Comment:
`walletConnectAccountSessions.first().isNotEmpty()`

Reply:
It's number so I can't use isNotEmpty but I changed walletConnectAccountSessions to walletConnectAccountSessionCount to make clearer the responsibility